src/components/AlertsMenu/AlertsMenuWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
```

Comment:
this doesn't use any client side APIs so could be a server component

Reply:
I get an error when I make server component, because it uses useSidebar from the context